core/compute/compute.go

```diff
@@ -148,13 +148,7 @@ func DoInstancePull(params model.ImageParams, progress *progress.Progress, docke
 		return types.ImageInspect{}, errors.Wrap(err, constants.DoInstancePullError+"failed to pull image")
 	}
 
-	if len(params.Tag) > 0 {
-		repoTag := fmt.Sprintf("%s:%s", dockerImage.Repo, dockerImage.Tag+params.Tag)
-		if err := dockerClient.ImageTag(context.Background(), dockerImage.UUID, repoTag); err != nil && !client.IsErrImageNotFound(err) {
-			return types.ImageInspect{}, errors.Wrap(err, constants.DoInstancePullError+"failed to tag image")
-		}
-	}
-	inspect, _, err2 := dockerClient.ImageInspectWithRaw(context.Background(), dockerImage.UUID)
```

Comment:
assume this should be removed because dockerImage.Repo is no longer parsed in cattle.